app/src/main/java/co/develhope/chooseyouowncocktail_g2/adapter/DrinkCardAdapter.kt

```diff
@@ -75,5 +73,14 @@ class DrinkCardAdapter(
         return beerListForAdapter.size
     }
 
+}
+
```

Comment:
sposterei questa funzione in un file dove ci saranno solo le ext fun per le imageview